docs-markdown/src/controllers/moniker-controller.ts

```diff
@@ -0,0 +1,90 @@
+"use strict";
+
+import { Range, TextEditor, window } from "vscode";
+import { insertContentToEditor, setCursorPosition, isMarkdownFileCheck, noActiveEditorMessage } from "../helper/common";
+import { isCursorInsideYamlHeader } from "../helper/yaml-metadata";
+import { sendTelemetryData } from "../helper/telemetry";
+
+const telemetryCommand: string = "insertMoniker";
+let sign: string = "";
```

Comment:
Please declare this within the `insertMoniker` function to reduce global variables.